src/client/src/app/services/auth.service.ts

```diff
@@ -1,17 +1,50 @@
-import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { IUser } from "../model/IUser";
 
 @Injectable({
-	providedIn: 'root',
+	providedIn: "root",
 })
 export class AuthService {
 	constructor(private http: HttpClient) {}
 
+	apiPath: string = "http://localhost:5000/";
```

Comment:
Sollte in eine Umgebungsvariable ausgelagert werden, müssen wir aber mal noch schauen wie wir das handlen